resources/analysisTools/sophiaworkflow/fusionCandidates.py

```diff
@@ -29,59 +29,93 @@
 import fileinput
 import re
 import itertools
-from sys import stderr
+import sys
+
 
 MAXDISTANCE = 2000000
 
 
-def natural_sort(l):
+def natural_string_sort(strings: list) -> list:
```

Comment:
Note that the type annotation `: list` in Python 3 is not binding, which means Python is not checking it, but it is helpful in IntelliJ, where type mismatches are checked and highlighted!